### PR TITLE
F 469 aggregate delegate conversations

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #469

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
It's working fine on my local
Screen Shot 1400-01-19 at 13 06 40


Screen Shot 1400-01-19 at 13 06 28

File: src/utils/conversationCreator.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    conversationId: similarDelegation._id,
+    payments: [newPayment],
+  });
+}
+
+const createDelegatedConversation = async (
+  app,
+  { milestoneId, donationId, txHash, payment, parentDonations, actionTakerAddress },
+) => {
+  const similarDelegation = await findSimilarDelegatedConversation(app, {
```
Comment (aminlatifi) [inline src/utils/conversationCreator.js:53]: `findSimilarDelegatedConversation` keeps a projection parameter, it's not passed so its default value `{}` is considered by function. So `similarDelegation` would not have payments property which will be used in `updateSimilarDelegatedConvesationPayments` function few lines below (line 59)

**Review by aminlatifi — COMMENTED** [review]
@mohammadranjbarz Please make me sure every feature is tested by you. I know you probably have tested, but double test is worthful

**Comment by aminlatifi** [comment]
Seems OK, thanks @mohammadranjbarz

**Pull request merged**